bokehjs/src/coffee/renderer/glyph/line.coffee

```diff
@@ -33,6 +33,131 @@ define [
       if drawing
         ctx.stroke()
 
+    _hit_point: (geometry) ->
+      ### Check if the point geometry hits this line glyph and return an object
+      that describes the hit result:
+
+        Args:
+          * geometry (object): object with the following keys
+            * vx (float): view x coordinate of the point
+            * vy (float): view y coordinate of the point
+            * type (str): type of geometry (in this case it's a point)
+
+        Output:
+          Object with the following keys:
+            * 0d (bool): whether the point hits the glyph or not
+            * 1d (array(int)): array with the indices hit by the point
+
+      ###
+      [vx, vy] = [geometry.vx, geometry.vy]
+      x = @renderer.xmapper.map_from_target(vx)
+      y = @renderer.ymapper.map_from_target(vy)
+      [x0, x1] = [x-1, x+1]
+      [y0, y1] = [y-1, y+1]
+
+      result = {
+        '0d': false,
+        '1d': []
+      }
+
+      for i in [0...@x.length-1]
+        console.log x0, y0, x1, y1, @x[i], @y[i], @x[i+1], @y[i+1]
+        res = @check_intersect(x0, y0, x1, y1, @x[i], @y[i], @x[i+1], @y[i+1])
```

Comment:
Yes.. the idea was that the span woud have a sort "thickness" ... a sort of tollerance so that you can be "close" enough to the line for the tooltip to show. We can remove that.
What's your suggestion for `hit_point` instead?